### ISSUE TITLE
JSON::showOnIndex() should be compatible with

### ISSUE THREAD

**Issue by sirikkoster** [64robots/nova-fields#116]
Not compatibel with the latest version of Nova

```
Declaration of R64\\NovaFields\\JSON::showOnIndex() should be compatible with Laravel\\Nova\\Fields\\FieldElement::showOnIndex($callback = true)
```

https://github.com/64robots/nova-fields/blob/7e6c7047af9f44ec6ce0ae76d5e7677610cd1bff/src/JSON.php#L103

**Comment by pehbehbeh** [comment]
Same issue here. New Method looks like this:

```php
    /**
     * Specify that the element should be hidden from the index view.
     *
     * @param  \Closure|bool  $callback
     * @return $this
     */
    public function showOnIndex($callback = true)
    {
        $this->showOnIndex = $callback;

        return $this;
    }
```

**Comment by pehbehbeh** [comment]
I am not quire sure, why the method is overwritten in the `JSON` class. It seems redundant without any custom logic... `JSON` extends `Field` extends  `FieldElement` which already has the `$showOnIndex` member and `showOnIndex` method.

It can be removed from the `JSON` class or am I missing something?

**Comment by disitec** [comment]
@pehbehbeh you are totally right. It is an unnecessary feature dragged from the beginning of the project. FIxed in 0.11.1.

**Comment by pehbehbeh** [comment]
Thanks!
